tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +16,128 @@
+                  result_field='EDUCATION_LEVEL', result_function=validators.or_validators(
+                      validators.isInStringRange(1, 16, 2),
+                      validators.isInStringRange(98, 99)
+                      )
+                  ),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='CITIZENSHIP_STATUS', result_function=validators.isInLimits(1, 2)
+                  ),
+              validators.validate__FAM_AFF__HOH__Count_Fed_Time(),
+              validators.if_then_validator(
+                  condition_field='DATE_OF_BIRTH', condition_function=validators.olderThan(18),
+                  result_field='REC_OASDI_INSURANCE', result_function=validators.isInLimits(1, 2)
+                  ),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='REC_FEDERAL_DISABILITY', result_function=validators.isInLimits(1, 2)
+                  ),
+                ],
           fields=[
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.dateMonthIsValid()]),
               Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isAlphaNumeric()]),
               Field(item="14", name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isInLimits(1, 5)]),
               Field(item="15", name='DATE_OF_BIRTH', type='number', startIndex=20, endIndex=28,
-                    required=True, validators=[]),
-              EncryptedField(decryption_func=tanf_ssn_decryption_func, item="16", name='SSN', type='string',
-                             startIndex=28, endIndex=37, required=True, validators=[]),
+                    required=True, validators=[validators.isLargerThan(0)]),
+              TransformField(transform_func=tanf_ssn_decryption_func, item="16", name='SSN', type='string',
+                             startIndex=28, endIndex=37, required=True, validators=[validators.isNumber()],
+                             is_encrypted=False),
               Field(item="17A", name='RACE_HISPANIC', type='number', startIndex=37, endIndex=38,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isInLimits(1, 3)]),
```

Comment:
```suggestion
                    required=True, validators=[validators.isInLimits(0, 3)]),
```

all of the race/ethnicity fields can have a value of `0`